subsys/bluetooth/mesh/Kconfig

```diff
@@ -378,15 +378,15 @@ config BT_MESH_IVU_DIVIDER
 config BT_MESH_TX_SEG_MSG_COUNT
 	int "Maximum number of simultaneous outgoing segmented messages"
 	default 1
-	range 1 255
+	range 0 255
```

Comment:
I guess it is mandatory to emphasize that if a customer chooses 0 then it is not BLE Mesh compatible anymore. It has to be added here and for `BT_MESH_RX_SEG_MSG_COUNT`
Segmentation\reassemblying is a mandatory feature of specification. Even more, some access messages use segmented frames if they consist of only 1 segment to enable reliable communication on the transport layer including transport Ack. This for mandatory for Remote Provisioning in v1.1 for example.

A customer should clearly understand that choosing this option he\she cannot expect the correct functioning of the Mesh network.

Reply:
Yes, it is necessary to add more explanatory information.